package.json

```diff
@@ -13,13 +13,14 @@
     "test": "gulp test",
     "test:coverage": "gulp coverage",
     "start": "gulp dev",
-    "postinstall": "bower install",
     "precommit": "gulp lint",
     "build": "gulp build"
   },
   "dependencies": {
     "kibiutils": "sirensolutions/kibiutils#1.4.0",
-    "kibi-icons": "sirensolutions/kibi-icons#1.0.0"
+    "kibi-icons": "sirensolutions/kibi-icons#1.0.0",
+    "lodash": "3.10.1",
```

Comment:
I think lodash is not needed as it is already present in kibana/kibi
Could you try remove it and test if everything still works